examples/push/BUILD.bazel

```diff
@@ -27,10 +27,19 @@ oci_image_index(
     ],
 )
 
+stamped_tags(
```

Comment:
naming suggestion: `stamp_tags`  usually our rules use verb names instead of a version specific to a tense.

Reply:
okay sure, matters less now that it's in the example